auslib/test/admin/views/test_rules.py

```diff
@@ -678,22 +678,22 @@ def setUp(self):
         super(TestRuleScheduledChanges, self).setUp()
         dbo.rules.scheduled_changes.t.insert().execute(
             sc_id=1, scheduled_by="bill", data_version=1, base_rule_id=1, base_priority=100, base_version="3.5", base_buildTarget="d",
-            base_backgroundRate=100, base_mapping="b", base_update_type="minor", base_data_version=1,
+            base_backgroundRate=100, base_mapping="b", base_update_type="minor", base_data_version=1, change_type="update",
```

Comment:
I know it's a pain in the butt, but please add change_type to all of the history tables as well - otherwise this test data is not quite representative of production.